sql/patch-schema-2.sql

```diff
@@ -76,3 +76,58 @@ ALTER TABLE `userfilters` CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unico
 ALTER TABLE `users` CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
 ALTER TABLE `userScores_mv` CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
 ALTER TABLE `wishlists` CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+
+
+--
+-- Table structure for table `playertimes`
+--
+
+DROP TABLE IF EXISTS `playertimes`;
+CREATE TABLE playertimes (
+  id INT AUTO_INCREMENT,
+  gameid VARCHAR(32) NOT NULL,
+  userid VARCHAR(32) NOT NULL,
+  time_in_minutes INT(5) unsigned not null,
+  PRIMARY KEY (id),
+  KEY `gameid` (`gameid`),
+  KEY `userid` (`userid`),
+  UNIQUE KEY `user_game` (`userid`, `gameid`)
```

Comment:
I think it makes more sense for the unique key to have the game first, to match the order of the columns.

And then there's no need for the KEY on `gameid`, since any queries on just `gameid` can reuse the index `gameid, userid`.